src/net_processing.cpp

```diff
@@ -515,6 +515,9 @@ struct Peer {
     /** Set of txids to reconsider once their parent transactions have been accepted **/
     std::set<uint256> m_orphan_work_set GUARDED_BY(g_cs_orphans);
 
+    /** Work queue of items requested by this peer **/
```

Comment:
2d9f2fca43aadcdda4d644cddab36dca88b40b97

To describe a member `//!` is [recommended](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-doxygen-compatible-comments):
```suggestion
    //! Work queue of items requested by this peer.
```

Reply:
As noted above, leaving this as is.